mod/messageboard/views/default/forms/messageboard/add.php

```diff
@@ -5,19 +5,25 @@
  * @package ElggMessageBoard
  */
 
-echo elgg_view('input/plaintext', array(
+
+echo elgg_view_field([
+	'#type' => 'plaintext',
```

Comment:
Consistency, I suppose. No biggie. To be honest I'd rather loose messageboard entirely :)